src/pages/blog/recent-featured-posts/_style.js

```diff
@@ -234,3 +232,35 @@ export const DotIcon = styled.img`
     margin: 2px 4px;
     color: var(--color-white);
 `
+
+export const MobileDotIcon = styled(DotIcon)`
+    height: 5px;
+`
+
+export const SmallArticleTopContent = styled(Flex)`
+    color: black;
+    height: 30px;
+`
+
+export const SmallArticleDateTimeMobile = styled(Flex)`
+    color: var(--color-grey-5);
+    width: 100%;
+    height: 20px;
+    font-size: 12px;
+    visibility: hidden;
```

Comment:
Are we sure we want visibility hidden here? It allocates space for the element as compared to display none that treats it as though the element is not there. Same for the other instances as well.